docs/code_coverage.md

```diff
@@ -0,0 +1,56 @@
+# Code Coverage
+
+You can generate code coverage report for your project using [Clang Source-based
+Code Coverage].
+
+
+## Build fuzz targets
+
+Code Coverage report generation requires a special build configuration to be
+used. In order to produce such build for your project, run:
+
+```bash
+python infra/helper.py build_image $project_name
+python infra/helper.py build_fuzzers --sanitizer=profile $project_name
+```
+
+
+## Establish access to GCS
+
+To get a good understanding of quality of the fuzz testing established for your
+project, code coverage should be generated by running fuzz targets against the
+corpus aggregated by OSS-Fuzz. The helper script will download the corpus
+automatically using `gsutil` tool. To check whether you have access to the
```

Comment:
I think it would be helpful to add a link to https://cloud.google.com/storage/docs/gsutil_install somewhere.

Reply:
Good point! Added.